jedi.inc

```diff
@@ -1935,6 +1936,7 @@
   {$DEFINE HAS_UNIT_PNGIMAGE}
   {$DEFINE HAS_UNIT_CHARACTER}
   {$DEFINE HAS_EXCEPTION_STACKTRACE}
+  {$DEFINE SUPPORTS_SCOPEDENUMS}
```

Comment:
This should not be in the RTL section but in the appropriate COMPILER section above because it's a feature supported by the compiler, not a method or unit provided by the RTL

Reply:
Thank you for your review and reply. I think I did it right in the next commit.
But as someone that don't contribute to JEDI often, please, keep guiding me.